source/io/module_eggShell.js

```diff
@@ -10,14 +10,6 @@ var assignEggShell;
         Module.eggShell = {};
         publicAPI.eggShell = {};
 
-        Module.eggShell.readJavaScriptRefNum = publicAPI.eggShell.readJavaScriptRefNum = function (valueRef) {
```

Comment:
Updated these functions to use the same error handling pattern as our other public eggShell functions below